apps/user_ldap/lib/LDAP.php

```diff
@@ -295,22 +307,30 @@ protected function invokeLDAPMethod() {
 			if ($this->isResultFalse($result)) {
 				$this->postFunctionCall();
 			}
+			if ($this->dataCollector !== null) {
+				$this->dataCollector->stopLastLdapRequest();
+			}
 			return $result;
 		}
 		return null;
 	}
 
-	/**
-	 * @param string $functionName
-	 * @param array $args
-	 */
-	private function preFunctionCall($functionName, $args) {
+	private function preFunctionCall(string $functionName, array $args): void {
 		$this->curFunc = $functionName;
 		$this->curArgs = $args;
 
+		if ($this->dataCollector !== null) {
+			$args = array_reduce($this->curArgs, static function (array $carry, $item): array {
+				$carry[] = !isResource($item) ? $item : '(resource)';
```

Comment:
```suggestion
				$carry[] = !$this->isResource($item) ? $item : '(resource)';
```
Same for the other one.